src/core/gltf2exporter/dracocompressor_p.cpp

```diff
@@ -151,6 +151,35 @@ QByteArray packedDataInBuffer(const QByteArray &inputBuffer,
     return outputBuffer;
 }
 
+QString semanticNameFromAttribute(const Qt3DRender::QAttribute &attribute)
+{
+    auto str = attribute.property("semanticName").toString();
+    if (!str.isEmpty()) {
+        return str;
+    } else {
+        const auto name = attribute.name();
```

Comment:
I guess you could make this more readable by creating a const QHash<QString, QString> and doing a .value lookup on it.

Reply:
ok, I'll refactor it in another commit since there is pretty similar code elsewhere